.buildkite/pipelines/lucene-snapshot/run-tests.yml

```diff
@@ -0,0 +1,60 @@
+steps:
+  - label: sanity-check
+    command: .ci/scripts/run-gradle.sh -Dbwc.checkout.align=true -Dorg.elasticsearch.build.cache.push=true -Dignore.tests.seed -Dscan.capture-task-input-files precommit
```

Comment:
Weren't we wrapping up a lot of this boilerplate into a `GRADLEW` env var in the prcommand hook script?

Reply:
Currently, it's this:

```
GRADLEW="./gradlew --parallel --scan --build-cache --no-watch-fs -Dorg.elasticsearch.build.cache.url=https://gradle-enterprise.elastic.co/cache/"
```

Maybe we can do a pass later and sort out what can be configured globally across all jobs?

Reply:
Sounds good.